src/main/webapp/components/result-export/result-export.js

```diff
@@ -114,9 +112,53 @@ const ResultExport = props => {
       <Button
         disabled={selectedFormat === '' ? true : false}
         fullWidth
-        onClick={() => {
+        onClick={async () => {
           const encodedTransformer = getExportFormatId(selectedFormat)
-          props.saveFile({ result, exportResult, encodedTransformer })
+          const cql = getResultSetCql(resultsToExport)
```

Comment:
frontend shouldn't have knowledge of cql. We should be passing in a filter tree and having graphql handle the conversion